test/torch_test_meta.py

```diff
@@ -159,60 +159,57 @@
 
     # test_indexing.py
     # TestIndexing
-    'test_setitem_expansion_error',  # doesn't raise
-    'test_multiple_byte_mask',
-    'test_empty_slice',
-    'test_byte_tensor_assignment',
-    'test_byte_mask',
-    'test_byte_mask_accumulate',
-    'test_bool_indices',
-    'test_trivial_fancy_out_of_bounds',  # doesn't raise
-    'test_boolean_assignment_value_mismatch',  # doesn't raise
+    'test_setitem_expansion_error',  # expecting a different runtime error
+    'test_multiple_byte_mask', # expecting a different runtime error
+    'test_empty_slice', # stride
+    'test_byte_tensor_assignment', # expecting a different runtime error
+    'test_byte_mask', # expecting a different runtime error
+    'test_byte_mask_accumulate', # expecting a different runtime error
+    'test_bool_indices', # expecting a different runtime error
     'test_index_getitem_copy_bools_slices',  # storage
     'test_getitem_scalars',  # storage
-    'test_empty_ndim_index',  # runtime error
+    'test_empty_ndim_index',  # expecting a different runtime error
+
+    # NumpyTests
+    'test_trivial_fancy_out_of_bounds',  # expecting a different runtime error
+    'test_boolean_assignment_value_mismatch',  # expecting a different runtime error
     'test_empty_tuple_index',  # storage
-    'test_empty_fancy_index',  # runtime error
+    'test_empty_fancy_index',  # expecting a different runtime error
     'test_ellipsis_index',  # storage
-    'test_broaderrors_indexing',  # runtime error
-    'test_broadcast_subspace',  # runtime error
-    'test_boolean_shape_mismatch',  # runtime error
-    'test_boolean_indexing_weirdness',  # runtime error
-    'test_boolean_indexing_weirdness_tensors',  # runtime error
+    'test_broaderrors_indexing',  # expecting a different runtime error
+    'test_broadcast_subspace',  # XLA doesn't support arange with negative step
+    'test_boolean_shape_mismatch',  # expecting a different runtime error
+    'test_boolean_indexing_weirdness',  # expecting a different runtime error
+    'test_boolean_indexing_weirdness_tensors',  # expecting a different runtime error
 
     # test_nn.py
     # TestNNDeviceType
-    'test_embedding_backward',  # uses sparse
+    'test_embedding_backward',  # sparse
     'test_embedding_dense_grad',  # slow
-    'test_EmbeddingBag_per_sample_weights_and_new_offsets', # wrong result
-    'test_batchnorm_grad',
-    'test_gumbel_softmax_xla',
-    'test_rnn_retain_variables',
-    'test_pool_invalid_size',  # doesn't raise
-    'test_nonlinearity_propagate_nan',
+    'test_EmbeddingBag_per_sample_weights_and_new_offsets', # UndefinedTensorImpl::_singleton
```

Comment:
`UndefiendTensorImpl::_singleton` is a very internal concept, we probably need to either fix this  or make it a better error message?